Makefile

```diff
@@ -55,3 +63,32 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+.PHONY: bench-deps
+bench-deps:
+	@echo "Installing benchstat tool"
+	rm -rf ${BENCH_PATH}
+	mkdir ${BENCH_PATH}
+	go clean -testcache
+	cd ${BENCH_PATH} && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	rm -rf ${BENCH_PATH}/perf
+
+.PHONY: bench
+bench:
+	@echo "Running benchmark tests in the current branch"
+	go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 \
+		| tee ${BENCH_FILE}
+	benchstat ${BENCH_FILE}
+
+.PHONY: bench-reference
+bench-reference:
+	@echo "Running benchmark tests from master for using results in bench-diff target"
+	cd ${BENCH_PATH} && rm -rf go-tarantool && git clone https://github.com/tarantool/go-tarantool.git
+	cd ${BENCH_PATH}/go-tarantool && go test ${BENCH_PATH}/go-tarantool/... -bench=. -run=^Benchmark -benchmem \
+ 		-benchtime=${DURATION} -count=${COUNT} 2>&1  | tee ${REFERENCE_FILE}
+	rm -rf ${BENCH_PATH}/go-tarantool
+
+.PHONY: bench-diff
+bench-diff:
```

Comment:
`bench-diff` shoud depend on `BENCH_FILE` and `REFERENCE_FILE`. Otherwise:

```
sergeyb@pony:~/sources/go/src/github.com/tarantool/go-tarantool$ make bench-diff 
Comparing performance between master and the current branch
benchstat /home/sergeyb/sources/go/src/github.com/tarantool/go-tarantool/bench/reference.txt /home/sergeyb/sources/go/src/github.com/tarantool/go-tarantool/bench/bench.txt
benchstat: open /home/sergeyb/sources/go/src/github.com/tarantool/go-tarantool/bench/bench.txt: no such file or directory
make: *** [Makefile:94: bench-diff] Error 1
```